kingpin/actors/aws/ecs.py

```diff
@@ -981,12 +976,15 @@ def _update_service(self, service_name, existing_service, override=None):
         Returns:
             Task Definition string that was updated or registered.
         """
-        deployment_configuration = self.service_definition[
-            'deploymentConfiguration']
         update_parameters = dict(
             cluster=self.option('cluster'),
             service=service_name,
-            deploymentConfiguration=deployment_configuration)
+            **self.service_definition)
+
+        if 'loadBalancers' in update_parameters:
+            del update_parameters['loadBalancers']
```

Comment:
If you care... `update_parameters.pop("loadBalancers", 1)` -- no need for the if statement.

Reply:
oh nice